presto-main/src/test/java/com/facebook/presto/operator/aggregation/noisyaggregation/sketch/TestSfmSketch.java

```diff
@@ -170,7 +170,7 @@ public void testMergePrivate()
 
         Bitmap refBitmap = sketch.getBitmap().clone(); // clone existing bitmap
         refBitmap.or(sketch2.getBitmap()); // take an OR with sketch2, for later comparison
-        sketch.mergeWith(sketch2, new TestingSeededRandomizationStrategy(3));
+        sketch.mergeWith(sketch2);
```

Comment:
This could be a problem, as it makes the test random (i.e., potentially flaky). It'd probably be best to keep this test seeded if possible.

Reply:
(I believe the other test is technically fine, though.)